### DISCUSSION TITLE
How to change FileUpload (Filepond) label?

### DISCUSSION THREAD

**Discussion by NicolasSacC** [filamentphp/filament#5057] [Help] [answered]
Hi!

I need to change Filepond's label (https://pqina.nl/filepond/docs/api/instance/properties/#labels) `labelIdle` property. Since I cannot directly access Filepond in my code, how can I change the value on the Filepond input?

I though about changing it using some JS code, but the object is in a Livewire component and not necessarily initiated... And I tried setting it in the php and it doesn't work.

**Comment by NicolasSacC** [comment, marked as answer]
For future reference, just hooking up on the Filepond:init event and changing the label once it is available did the job, but still looking for a "cleaner" way to do it?

**Comment by danharrin** [comment]
Try the `->placeholder('...')` method on the form component

**Reply by isaacdarcilla** [reply]
@danharrin 

When there is a network change, the placeholder gets reset to the default placeholder.

 ->placeholder('test')
